go/contacts/tests.py

```diff
@@ -61,6 +74,18 @@ def mkcontact(self, name=None, surname=None, msisdn=u'+1234567890',
             surname=unicode(surname or TEST_CONTACT_SURNAME),
             msisdn=unicode(msisdn), **kwargs)
 
+    def get_group_contact_keys(self, group):
+        contact_keys = []
+        contact_keys_page = group.backlinks.contact_keys()
+        while contact_keys_page is not None:
+            contact_keys.extend(contact_keys_page)
```

Comment:
Because I sometimes forget that I can just pass an iterable to `.extend()`. I'll fix those, thanks.